packages/transformers/elm/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@parcel/transformer-elm",
+  "version": "2.0.0-beta.1",
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "funding": {
+    "type": "opencollective",
+    "url": "https://opencollective.com/parcel"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/parcel-bundler/parcel.git"
+  },
+  "main": "lib/ElmTransformer.js",
+  "source": "src/ElmTransformer.js",
+  "engines": {
+    "node": ">= 10.0.0",
+    "parcel": "^2.0.0-alpha.3.1"
+  },
+  "dependencies": {
+    "@parcel/diagnostic": "^2.0.0-beta.1",
+    "@parcel/plugin": "^2.0.0-beta.1",
+    "command-exists": "^1.2.8",
+    "terser": "^5.2.1"
+  },
+  "peerDependencies": {
+    "elm": "^0.19.1-3 || 0.19.0-no-deps",
+    "elm-hot": "^1.1.4",
+    "node-elm-compiler": "^5.0.4"
```

Comment:
These should be removed now, they're unnecessary with `options.packageManager.require`.